SUBMODULE_VERSIONS.txt

```diff
@@ -4,16 +4,16 @@ daff5fead3fbe22c6fc58310ca3f49caf117f185 third_party/benchmark
 4fb0ff7069bd88ee85902f4d0bb62794e5f6d021 third_party/flatcc
 b1fbd33c06cdb0024c67733c6fdec2009d17b384 third_party/googletest
 88b845dee001723c4a0db1fe5477de735b6d3bb0 third_party/liburing
-4e9c6abfb047b811efd606ed42ed92ac5c66285d third_party/llvm-bazel
-720a828045e1095ec7bd69c84fa1cd9c431a3179 third_party/llvm-project
-3483f1653fc7cb3bfb3a4d1b463f3a651ecaa676 third_party/mlir-emitc
-d16860d26dffb8eabf0c8ab004c2d70d115dfc2f third_party/mlir-hlo
+fe670bc54a575329ab730f78ecaf3383933b7ff9 third_party/llvm-bazel
+6878be5dc3ec7031d0deec3e321310115bd71103 third_party/llvm-project
+e691ab9fc49d728f6632408d534b82cb3d137b3f third_party/mlir-emitc
```

Comment:
revert mlir-emitc change here (`google` branch should not be modifying that submodule)